src/algebra/lie_algebra.lean

```diff
@@ -175,28 +175,109 @@ end prio
 
 namespace lie_algebra
 
-/--
-A morphism of Lie algebras is a linear map respecting the bracket operations.
--/
-structure morphism (R : Type u) (L : Type v) (L' : Type v)
+set_option old_structure_cmd true
+/-- A morphism of Lie algebras is a linear map respecting the bracket operations. -/
+@[nolint doc_blame has_inhabited_instance]
```

Comment:
The zero morphism would be an example, right?

Reply:
Indeed it would, or the identity map. I wasn't really clear to me when we were supposed to brush aside `has_inhabited_instance` and when we should obey it. This is a great time to get that clarified. I'll add a commit with examples.

Reply:
Examples now added.